tf-psa-crypto/tests/suites/test_suite_psa_crypto_util.function

```diff
@@ -3,7 +3,7 @@
 #include <mbedtls/psa_util.h>
 /* END_HEADER */
 
-/* BEGIN_CASE depends_on:PSA_HAVE_ALG_SOME_ECDSA */
+/* BEGIN_CASE depends_on:MBEDTLS_PSA_UTIL_HAVE_ECDSA */
```

Comment:
#9384 has been replacing and we shouldn't be undoing that here. Please remove the changes to the test files (which were correct at the time you made them, btw, but development is moving fast these days).